dotnet/src/VectorDataIntegrationTests/CosmosNoSQLIntegrationTests/Filter/CosmosNoSQLBasicQueryTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq.Expressions;
+using CosmosNoSQLIntegrationTests.Support;
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Filter;
+using VectorDataSpecificationTests.Support;
+using Xunit;
+
+namespace CosmosNoSQLIntegrationTests.Filter;
+
+public class CosmosNoSQLBasicQueryTests(CosmosNoSQLBasicQueryTests.Fixture fixture)
+    : BasicQueryTests<string>(fixture), IClassFixture<CosmosNoSQLBasicQueryTests.Fixture>
+{
+    // CosmosDB supports ordering by multiple fields only when a composite index is created up-front:
```

Comment:
I think the sorting/OrderBy functionality should probably be covered in a different test suite, leaving this one to only deal with all the variants of filter expressions (that's already more than enough stuff to cover 😅 ). But if you'd rather move faster I'm happy to clean it up later.